ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -46,29 +33,55 @@ function set_test_provider() {
     # access.
     if [[ "${DS_IP_STACK}" != "v6" ]];
     then
-        export TEST_PROVIDER='\{\"type\":\"baremetal\"\}'
+        export TEST_PROVIDER='{"type":"baremetal"}'
     else
-        export TEST_PROVIDER='\{\"type\":\"baremetal\",\"disconnected\":true\}'
+        export TEST_PROVIDER='{"type":"baremetal","disconnected":true}'
     fi
 }
 
+function setup_proxy() {
+    # For disconnected or otherwise unreachable environments, we want to
+    # have steps use an HTTP(S) proxy to reach the API server. This proxy
+    # configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+    # environment variables, as well as their lowercase equivalents (note
+    # that libcurl doesn't recognize the uppercase variables).
+    if test -f "${SHARED_DIR}/proxy-conf.sh"
+    then
+        # shellcheck source=/dev/null
+        source "${SHARED_DIR}/proxy-conf.sh"
+    fi
+}
+
+function is_openshift_version_gte() {
+    printf '%s\n%s' "$1" "${DS_OPENSHIFT_VERSION}" | sort -C -V
+}
+
 case "${CLUSTER_TYPE}" in
 packet)
     # shellcheck source=/dev/null
     source "${SHARED_DIR}/packet-conf.sh"
     # shellcheck source=/dev/null
     source "${SHARED_DIR}/ds-vars.conf"
-    copy_test_binaries
+        
+    setup_proxy
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
 
     echo "### Checking release version"
-    if printf '%s\n%s' "4.8" "${DS_OPENSHIFT_VERSION}" | sort -C -V; then
+    if is_openshift_version_gte "4.8"; then
         # Set test provider for only versions greater than or equal to 4.8
         set_test_provider
 
         # Mirroring test images is supported only for versions greater than or equal to 4.8
         mirror_test_images
+
+        # Skipping proxy related tests ([Skipped:Proxy]) is supported only for version  greater than or equal to 4.10
+        # For lower versions they must be skipped manually
+        if ! is_openshift_version_gte "4.10"; then
+            TEST_SKIPS="${TEST_SKIPS}
```

Comment:
That means, we will not add serial-ipv6 and serial-compact for less than 4.10. I have never thought we would add them for <4.10 anyway.

Reply:
Do you maybe mean 4.8? This block is required untile the[ proxy skipping feature](https://github.com/openshift/origin/blob/d85394b6765779164e76ca3c3f66dc827fcaf887/test/extended/util/cluster/cluster.go#L214-L220) will not be backported for 4.9 and 4.8 in openshift/origin